contrib/tree-sitter-epics-nice-print.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
```

Comment:
Some system have bash installed elsewhere. Only `/usr/bin/env` (or `/bin/sh` which is not necessarily bash) are guaranteed by POSIX.

```suggestion
#!/usr/bin/env bash
```